scripts/ci_e2e_test.sh

```diff
@@ -220,6 +220,23 @@ test_controller_image() {
     --aws-region=${AWS_REGION} \
     --aws-vpc-id=${cluster_vpc_id} \
     --controller-image=${CONTROLLER_IMAGE_NAME}
+
+  AWS_ACCOUNT_ID=$(aws sts get-caller-identity --region ${AWS_REGION} --query Account --output text)
```

Comment:
we can put these at top of this file, and use
`ginko -v -r test/e2e/` (which will use invoke both test).

Reply:
sounds good